<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuckItChatClass</class>
 <widget class="QMainWindow" name="FuckItChatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FuckItChat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>447</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>3</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOkie">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>3</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Okie</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
   </widget>
   <addaction name="menuConnect"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FuckItChat.qrc"/>
 </resources>
 <connections/>
</ui>
